src/drawDirective.coffee

```diff
@@ -25,12 +25,13 @@ angular.module('ui-leaflet')
     scope: false
     replace: false
     require: ['leaflet']
-    controller: ($scope) ->
+    controller: ['$scope', ($scope) ->
```

Comment:
Interesting; I guess ngAnnotate does not catch those.

https://github.com/angular-ui/ui-leaflet-draw/blob/master/gulp/tasks/build.coffee#L41

Wonder if a ngAnnotate bump would help.